### ISSUE TITLE
Used grade Cultured Bioware can be bought

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#100]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/26_

----

```
What steps will reproduce the problem?
1. When creating character, You can select and actually buy Used grade Cultured 
Bioware

What is the expected output? What do you see instead?
You shouldn't be able to buy used bioware

What version of the product are you using? On what operating system?
0.0.5.139, Windows 7

Please provide any additional information below.
n/a

```

Original issue reported on code.google.com by `pzagdan...@gmail.com` on 23 Nov 2014 at 2:12

**Comment by chummer5a** [comment]
Deltaware is also available at chargen. Need to alter references to Second-Hand to Used. Should mostly sort it out.

**Comment by chummer5a** [comment]
Resolved in 5.167.
